tests/helm_test.go

```diff
@@ -0,0 +1,56 @@
+// +build k8s
+
+/*
+ * Copyright (C) 2018 IBM, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package tests
+
+import (
+	"testing"
+
+	"github.com/skydive-project/skydive/tests/helper"
+)
+
+const (
+	chartPath = "../contrib/charts"
+	relName   = "skydive"
+)
+
+func TestHelmInstall(t *testing.T) {
```

Comment:
what is the goal/scope of this test ? it's not directly related (out of scope) to skydive 
it's just testing if helm is installed correctly and working, that we assume it is in functional test

Reply:
It attempts to install skydive via helm chart (on the minikube environment), this was just a minimal test which (in future) should be extended to test that skydive (after installed by helm) is actually functioning correctly.

Reply:
Skydive functional tests is not designed for these tests
Please move this patch in a new scripts/ci/run-helm-tests.sh and run only inside the skydive functional k8s test (TEST_PATTERN=...)
We will trigger it after the docker job

Reply:
ok, I have separated helm from k8s as:
 - k8s  covers the k8s probe (ability of skydive to understand the topology/flow of a k8s cluster).
- helm coverts deployment of skydive onto a k8s cluster (after which we may want to check various functionality is working correctly)

Reply:
tests/helm_test.go should "check various functionality is working correctly"
helm delete / install  are jobs must be done in scripts/ci/run-k8s-tests.sh

Reply:
file in tests/test_*.go are tests to testing skydive, not for testing helm packaging